sphinx/util/__init__.py

```diff
@@ -612,3 +612,8 @@ def status_iterator(iterable, summary, color="darkgreen", length=0, verbosity=0,
         yield item
     if l > 0:
         logger.info('')
+
+
+def html5_writer_available():
```

Comment:
Personally, I'd like to name the functions which return a boolean value to `is_*()`. What do you think?

In addition, please add type annotation.
For this function, it's better to annotate like following:
```
def html5_writer_available():
    # type: () -> bool
```